keps/2724-topology-aware-scheduling/README.md

```diff
@@ -762,6 +929,39 @@ We recommend the BestFit algorithm for most of use cases, however we give more f
 Based on the collected feedback we will introduce TAS configuration that would allow user to select the desired algorithm.
 Eventually we'll remove the feature as they will be no longer need when we implement API for TAS configuration.
 
+### Two-level scheduling
+In consideration of a [Story 5](#story-5) a two-level scheduling is introduced. 
+We introduce a notion of PodSet Chunk, which is a subset of PodSet. All chunks
+have the same size and evenly divide the PodSet. To simplify implementation we 
+are allowing only for a "required" topology for chunk. The requested topology 
+level for chunks has to be on the same or below level of the main topology.
+
+Requesting chunk topology changes the way algorithm chooses domains. In all 
+modes in [Computing the assignment](#computing-the-assignment) algorithm greedily
+selects domains based on the free capacity (either most or least). However, with
+chunks it considers the amount of chunks that can fit into a domain as free 
+capacity, but if that number is equal then it prioritizes domains with fewer 
+remaining resources.
+
+#### Example
+Consider a rack with nodes that can accommodate 6, 5, 4, 3, and 2 pods respectively. 
+The required topology for chunk is "host" and the podset chunk size is 2.
+
+As an example let's analyze the assignment based on the size of a podset and selected 
+mode. See the table below where the numbers are pods assigned to a particular node.
+The header for columns dedicated to nodes correspond node's initial capacity.
+
+| mode                   | PodSet size | 6 | 5 | 4 | 3 | 2 |
+| -----------------------| ----------- | - | - | - | - | - |
+| BestFit                | 12          | 6 | . | 4 | . | 2 |
```

Comment:
I think it's worth noting that even with the `BestFit` algorithm there is no guarantee that there won't be any leftover capacity in the selected nodes, as there is already some rounding in the assumption nodes can accomodate an integer number of pods. The `BestFit` algorithm excellently mitigates resource fragmentation, and it's no doubt it's the way to go, but let's just keep in mind even that has some limits, and we won't mitigate fragmentation completely.

Reply:
I've added a one disclaimer sentence that the "tight fit" I'm referring to does not guarantee no dangling resources.